src/als/stack.py

```diff
@@ -335,10 +335,10 @@ def _find_transformation(self, image: Image):
                 matches_count = len(matches[0])
                 _LOGGER.debug(f"image matched features count : {matches_count}")
 
-                if matches_count < _MINIMUM_MATCHES_FOR_VALID_TRANSFORM:
+                if matches_count < minimum_matches_for_valid_transform:
                     _LOGGER.warning(f"Found transformation but matches count is too low : "
-                                    f"{matches_count} < {_MINIMUM_MATCHES_FOR_VALID_TRANSFORM}. "
-                                    "Discarding transformation")
+                                  f"{matches_count} < {minimum_matches_for_valid_transform}. "
```

Comment:
please fix identation :)

We try and comply to PEP8. And for this specific case, please see https://stackoverflow.com/questions/15435811/what-is-pep8s-e128-continuation-line-under-indented-for-visual-indent

Reply:
Fixed